src/options.cpp

```diff
@@ -1229,6 +1229,13 @@ void options_manager::add_options_general()
 
     add_empty_line();
 
+    add( "PREVENT_SAVE_SCUM", "general", translate_marker( "Prevent save scumming on death" ),
+         translate_marker( "If true, your character will be deleted immediately on death. Only enable if you wish to stay true to one life one death." ),
```

Comment:
Honestly I'd remove the `Only enable...` part. I think there' so need to tell players when or why they must enable the option.

Reply:
I will remove that part, and also set the default to true as per your recommendation above